wenet/bin/train.py

```diff
@@ -194,10 +193,11 @@
     if distributed:
         assert (torch.cuda.is_available())
         # cuda model is required for nn.parallel.DistributedDataParallel
-        model.cuda()
+        torch.cuda.set_device(args.rank)
```

Comment:
Maybe it should be `args.gpu`.

Reply:
Logically it should be args.gpu, but i failed with args.gpu but succeed with args.rank.....

Reply:
https://github.com/espnet/espnet/blob/master/espnet2/train/trainer.py#L229

for reference

Reply:
what if we use multiple GPUs on multiple machine?